event_routing_backends/models.py

```diff
@@ -97,13 +174,28 @@ class RouterConfiguration(ConfigurationModel):
         )
     )
 
+    route_url = models.CharField(
+        max_length=255,
+        verbose_name=_('Route url'),
+        default="",
```

Comment:
if blank is false then this default value does not make sense here

Reply:
Done